esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -81,7 +83,13 @@ def _create_cube(data, src_cube, mesh_dim, mesh, grid_x, grid_y):
 
     new_cube = iris.cube.Cube(data)
 
-    # TODO: add coords and metadata.
+    new_cube.add_dim_coord(grid_x, mesh_dim + 1)
+    new_cube.add_dim_coord(grid_y, mesh_dim)
```

Comment:
The dimension ordering is set so that `grid_y` is explicitly before `grid_x`. This will be the case even if the dimension ordering is the other way around in the target cube. The ability to be preserve the target cube dimension ordering may be added in future.

Reply:
@stephenworsley  thanks for making the changes we discussed. could you make this comment just a tad more descriptive of what would need to be done?  i.e. this is the specific case that will work for 2D, but needs to be reworked for 3D.  Thanks!